verification/application/services/verification_manager.py

```diff
@@ -134,6 +134,19 @@ def callback(self, verification_id, verification_details):
         self._ack_verification(verification)
         return {}
 
+    def slack_callback(self, entity_id, verification_details):
```

Comment:
this duns_no verification callback, slack is a way it is doing so,Please rename it